unfoldingWord/translationCore
# Styling for Greek popover

#### This pull request addresses:

Adjust padding on popover so scroll bar does not show up



#### How to test this pull request:

Check to make sure there is no scrollbar on the popover and the definition in line two should be indented.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1090)
